optuna/trial/_base.py

```diff
@@ -11,6 +11,9 @@
 from optuna.distributions import CategoricalChoiceType
 
 
+_SUGGEST_INT_POSITIONAL_ARGS = ["self", "name", "low", "high", "step", "log"]
```

Comment:
Could you make only `step` and `log` positional? Currently `convert_positional_args` does not support partial conversion, but you can use `inspect.signature` to achieve this.

Reply:
Hi sorry for the late response:(
The variable name was actually misleading because these are "previous" positional arguments, but not the current positional arguments.
Plus, since we do not have any non-positional arguments in `suggest_int` before this PR, I would like to simply change the variable name and let the `convert_positional_args` stay as it does.
If necessary, I can also make another PR for this.
How do you like it? (@not522)

Reply:
Oh, I'm sorry, it's my mistake. I wanted to say "make only `step` and `log` keyword-only".

Reply:
Certainly!
Then I will change the code so that inappropriate args usages will be warned!